packages/hebao_v1/contracts/modules/core/ForwarderModule.sol

```diff
@@ -65,20 +70,8 @@ abstract contract ForwarderModule is BaseModule
         public
         view
     {
-        // Since this contract is a module, we need to prevent wallet from interacting with
-        // Stores via this module. Therefore, we must carefully check the 'to' address as follows,
-        // so no Store can be used as 'to'.
-        require(
-            (to != address(this)) &&
-            controllerCache.moduleRegistry.isModuleRegistered(to) ||
-
-            // We only allow the wallet to call itself to addModule
-            (to == from) &&
-            data.toBytes4(0) == Wallet.addModule.selector ||
+        verifyTo(to, from, data);
```

Comment:
if `to` is `address(this)`, shall we check the function must be `batchCall`?
Inside `batchCall`, shall we check `to` can never be `address(this)`?

I assume the expected use case is to use `executeMetaTx` to invoke `batchCall`, not the other way around.

Reply:
Can do. I don't think there can be any problems not doing those checks, but can't hurt to limit how it can be used to make it easier to think about.

Reply:
Lets be cautious by adding these extra checks.